lib/hashie/extensions/coercion/active_model.rb

```diff
@@ -0,0 +1,85 @@
+require 'active_model/type'
+
+module Hashie
+  module Extensions
+    module Coercion
+      module ActiveModel
+        # Symbol is the one glaring type omission. Define it if it is not
```

Comment:
Is this a version thing? I mean is symbol going to be defined in the future? Feels weird to be defining a type within ActiveModel here, it should be external to here and probably not be in the ActiveModel namespace, but in Hashie's.